v3/tutorials/09-cumulus/a-start-relay/index.mdx

```diff
@@ -124,9 +124,9 @@ RUSTFLAGS="-C codegen-units=1" cargo build --release
 
 This tutorial has been tested on:
 
-- [Polkadot `v0.9.22`](https://github.com/paritytech/polkadot/tree/release-v0.9.22)
-- [Substrate Parachain Template `polkadot-v0.9.22`](https://github.com/substrate-developer-hub/substrate-parachain-template/tree/polkadot-v0.9.22)
-- [Polkadot-JS Apps `v0.115.3-9`](https://github.com/polkadot-js/apps/commit/2174383a1aefd293f5122cd9b1ae6ce2c28a2a2d).
+- [Polkadot `v0.9.23`](https://github.com/paritytech/polkadot/tree/release-v0.9.23)
+- [Substrate Parachain Template `polkadot-v0.9.23`](https://github.com/substrate-developer-hub/substrate-parachain-template/tree/polkadot-v0.9.23)
+- [Polkadot-JS Apps `v0.115.3-49`](https://github.com/polkadot-js/apps/commit/18d95a4a61cdd31ec64b034ce07fafa46dd87258).
```

Comment:
@brunopgalvao this is the change you need to get from the Apps UI repo, version matches what you see on the top right of the apps ui, the latest beta skip-ci release is usually it ;)